js/index.js

```diff
@@ -1,144 +1,252 @@
+    entries: [
+      { name: "Relational Databases", rating: 4 },
+      { name: "ETL Pipelines", rating: 3 },
+      { name: "Google Cloud / AWS", rating: 3 },
+      { name: "Redis & RabbitMQ", rating: 2 },
+      { name: "Docker/Kubernetes", rating: 2 },
+    ],
+  },
+  {
+    title: "Non-Engineering Skills",
+    entries: [
+      { name: "UI/UX Design", rating: 3 },
+      { name: "Project Management", rating: 2 },
+      { name: "A/B Experiment Design", rating: 2 },
+      { name: "Copy Writing", rating: 2 },
+    ],
+  },
 ];
 
+const EXPERIENCE = [
+  {
+    title: "Loom, Inc.",
+    link: "https://loom.com",
+    metadata: ["Senior Software Engineer", "September 2021 - June 2022"],
+    tasks: [
+      "Led feature development for a high-traffic Slack integration using queues and rate-limiting",
+      "Collaborated with data analysts, UI designers, and product managers to drastically expand product presence for a chrome extension with >100,000 daily active users",
+      "Conducted A/B experiments to test our assumptions on user behavior and make more informed product decisions",
+    ],
+  },
 
-function initSkills(container){
-	for( var i in skillSets ){
-		var skillSet = skillSets[i];
+  {
+    title: "Wakebird",
+    link: "https://wakebird.com",
+    metadata: ["Co-Founder", "January 2019 - Present"],
+    tasks: [
+      "Led a two-person team to design, build, and sell a web application for organizing dragon boat racing teams",
+      "Mentored a junior engineer on code review, sprint planning, node.js, and React",
+      "Synthesized interviews with athletes and coaches into meaningful feature design specs",
+      "Taught athletes and coaches how to use Wakebird, and solicited feedback to get to the bottom of customer needs",
+      "Reviewed bookkeping ledger and tax remittance on a quarterly basis",
+    ],
+  },
 
-		var columnDiv = $("<div/>",{
-			"class":"column"
-		}).appendTo(container);
+  {
+    title: "Flatiron Health",
+    link: "https://flatiron.com",
+    metadata: ["Senior Software Engineer", "June 2019 - August 2021"],
+    tasks: [
+      "Drafted and built the infrastructure for a machine-learning-based product that predicts patient ER visits for oncology nurses",
+      "Refactored and maintained complex Extract, Transform & Load (ETL) pipelines with Apache Airflow for quarterly delivery of abstracted cancer data",
+      "Designed UI and wrote the code for a research project tracking dashboard that served as a communication platform between academic researchers and Flatiron clinical data analysts",
+    ],
+  },
 
-		$("<h3/>",{
-			text: skillSet.title
-		}).appendTo(columnDiv);
+  {
+    title: "Knewton",
+    link: null,
+    metadata: ["Software Engineer II", "November 2018 - March 2019"],
+    tasks: [
+      "Investigated slow queries to solve slowdowns on a Django-based Content Management System (CMS).",
+      "Quickly picked up Python, Typescript, and Phabricator concepts",
```

Comment:
That's great, shows you are a fast learner. This is expected of any engineer at any company.
What are you really trying to communicate with this point? What makes your contributions to this team special?

Reply:
I'm trying to communicate that I basically didn't do much else at Knewton because I wasn't there for long and the code was highly engineered.